code/modules/projectiles/projectile/megabuster.dm

```diff
@@ -6,12 +6,12 @@
 	damage_type = BURN
 	hitsound = 'sound/weapons/sear.ogg'
 	hitsound_wall = 'sound/weapons/effects/searwall.ogg'
-	lefthand_file = 'modular_citadel/icons/mob/citadel/guns_lefthand.dmi'
-	righthand_file = 'modular_citadel/icons/mob/citadel/guns_righthand.dmi'
+	lefthand_file = 'icons/mob/citadel/guns_lefthand.dmi'
```

Comment:
Move all the states of this file to 'icons/mob/inhands/weapons/guns_lefthand.dmi', damnit.